app/setup_handlers.go

```diff
@@ -40,7 +82,7 @@ func SetupHandlers(app *App) {
 	}
 	if upgradeInfo.Name == releaseVersion && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
 		storeUpgrades := storetypes.StoreUpgrades{
-			Added: []string{authoritytypes.ModuleName, lightclienttypes.ModuleName},
+			Added: []string{authoritytypes.ModuleName, lightclienttypes.ModuleName, consensustypes.ModuleName, crisistypes.ModuleName},
```

Comment:
Should crisis be added here ?

Reply:
Hmm , checked on develop the crisis module is already present . 
https://github.com/zeta-chain/zeta-node/blob/022ea57e25b4e0cb338b99bbd1c9e235f41ae954/app/app.go#L514-L514

Adding it again might result in a consensus version mismatch because the initial version has already been set . I am not sure though if there is a special reason for adding crisis again , the upgrade test might help when we run it